src/providers/trusted_service/context/key_management.rs

```diff
@@ -91,6 +92,12 @@ impl Context {
         self.send_request_with_key(ExportPublicKeyIn::default(), id)
     }
 
+    /// Export the key given its ID.
+    pub fn export_key(&self, id: u32) -> Result<Vec<u8>, Error> {
+        info!("Handling ExportKey request");
+        self.send_request_with_key(ExportKeyIn::default(), id)
```

Comment:
This is quite cool, didn't realise `prost` is able to do this. Wouldn't it work for `GenerateRandomOut` as well?